synapse/storage/state.py

```diff
@@ -356,6 +356,85 @@ def get_member_split(self) -> Tuple["StateFilter", "StateFilter"]:
 
         return member_filter, non_member_filter
 
+    def approx_difference(self, subtrahend: "StateFilter") -> "StateFilter":
+        """
+        Returns a state filter which represents `self - subtrahend`.
+
+        The resultant state filter MUST admit all state events that are admitted
+        by only this filter (`self`) and not `subtrahend`. (1)
```

Comment:
Not sure what the `(1)` refers to here.

Reply:
thanks, it was left over from a prior draft where I referred to that property as (1)